plugins/sudo/sudo.plugin.sh

```diff
@@ -0,0 +1,14 @@
+#! bash oh-my-bash.module
+
+add_sudo() {
+  local prefix suffix
+  prefix="${READLINE_LINE:0:$READLINE_POINT}"
+  suffix="${READLINE_LINE:$READLINE_POINT}"
+  if [[ -z $prefix ]] && [[ -z $sufix ]]; then
+    prefix="$(fc -ln -1 | awk '{$1=$1};1')"
```

Comment:
What does `awk '{$1=$1};1'` do? If you would like to just strip the beginning whitespaces, could you use `sed 's/^[[:space:]]\{1,\}//'`?

Reply:
replaced awk with sed

Reply:
Thanks!